File: 3275-minimum-number-of-pushes-to-type-word-i/minimum-number-of-pushes-to-type-word-i.py
class Solution:
    def minimumPushes(self, word: str) -> int:
        l=len(word)
        k=l//8
        j=l%8
        sum=0
        for i in range(1,k+1):
            sum+=8*i
        for i in range(j):
            sum+=k+1
        return sum

        

File: 3275-minimum-number-of-pushes-to-type-word-i/minimum-number-of-pushes-to-type-word-i.java
class Solution {
    public int minimumPushes(String word) {
        int l=word.length();
        int iter=l/8;
        int extra=l%8;
        int total=0;
        for(int i=0;i<=iter;i++){
            total+=8*i;
        }
        for(int j=0;j<extra;j++){
            total+=iter+1;
        }
     return total;   
    }
}